lib/discovery.js

```diff
@@ -88,13 +88,13 @@ function mixinDiscovery(MsSQL) {
 
     if (options.all && !owner) {
       sqlTables = paginateSQL('SELECT \'table\' AS "type", table_name AS "name", table_schema AS "owner"' +
-        ' FROM information_schema.tables', 'table_schema, table_name', options);
+        ' FROM INFORMATION_SCHEMA.tables', 'table_schema, table_name', options);
```

Comment:
The problem here is the "i" character. In turkish uppercase i is İ. i -> İ

No problem because other table/column names such as tables, table_schema, table_name dont contian "i" character.

For consistency, It would be correct to make uppercase all sql query.